From fc4049a8e33690d7e2617c234d98fe611e5ef15c Mon Sep 17 00:00:00 2001
From: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Date: Thu, 17 Mar 2016 16:09:33 +0900
Subject: [PATCH 731/909] thermal: renesas: Add emergency shutdown support in
 DT documentation

commit f483cf8f029c74e6d9e65e7525e8ffe7055b72e6 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Signed-off-by: Gaku Inami <gaku.inami.xw@bp.renesas.com>
(cherry picked from horms/renesas-bsp commit 831876202b1672e6eedf9efa9d2d95c5947b9cc9)
[jiada: Resolve conflict in Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt]
Signed-off-by: Jiada Wang <jiada_wang@mentor.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../bindings/thermal/rcar-gen3-thermal.txt    | 26 +++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt b/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt
index 24fc7e34dd6d..5d0ee4267f86 100644
--- a/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt
+++ b/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt
@@ -55,3 +55,29 @@ Example:
 			};
 		};
 	};
+
+* Emergency shutdown for R-CAR Gen3
+Emergency shutdown functionality provide the specific cooling mechanism
+for R-CAR Gen3. In case of high temperature(e.g over 100 degrees),
+it has the ability to rapidly cooling the system.
+
+Required property:
+- polling-delay:	The maximum number of milliseconds to wait between polls
+  Type: unsigned	when checking temperature for emergency shutdown.
+  Size: one cell
+
+- on-temperature:       This value indicate the emergency temperature and invoke
+  Type: unsigned	emergency shutdown functionality when exceeding this
+  Size: one cell	temperature.
+
+- off-temperature:	This value indicate the temperature to disable emergency
+  Type: unsigned	shutdown.
+  Size: one cell
+
+thermal-zones {
+	emergency {
+		polling-delay = <1000>;    /* milliseconds */
+		on-temperature = <110000>; /* millicelsius */
+		off-temperature = <95000>; /* millicelsius */
+	};
+};
-- 
2.17.1

